tensorflow_addons/layers/multihead_attention_test.py

```diff
@@ -25,45 +25,47 @@
 from tensorflow_addons.utils import test_utils
 
 
-@test_utils.run_all_in_graph_and_eager_modes
-class MultiHeadAttentionTest(tf.test.TestCase):
-    def test_output_shape(self):
-        batch_size = 10
-        num_heads = 8
-        head_size = 12
+def test_output_shape():
+    batch_size = 10
+    num_heads = 8
+    head_size = 12
 
-        q = tf.random.uniform((batch_size, 5, 9), dtype=np.float32)
-        k = tf.random.uniform((batch_size, 7, 11), dtype=np.float32)
-        v = tf.random.uniform((batch_size, 7, 13), dtype=np.float32)
+    q = tf.random.uniform((batch_size, 5, 9), dtype=np.float32)
+    k = tf.random.uniform((batch_size, 7, 11), dtype=np.float32)
+    v = tf.random.uniform((batch_size, 7, 13), dtype=np.float32)
 
-        mha = MultiHeadAttention(head_size=head_size, num_heads=num_heads)
+    mha = MultiHeadAttention(head_size=head_size, num_heads=num_heads)
 
-        output = mha([q, k, v])
+    output = mha([q, k, v])
 
-        self.assertEqual(output.shape[0], batch_size)
-        self.assertEqual(output.shape[1], q.shape[1])
-        self.assertEqual(output.shape[2], v.shape[2])
+    np.testing.assert_equal(output.shape[0], batch_size)
+    np.testing.assert_equal(output.shape[1], q.shape[1])
+    np.testing.assert_equal(output.shape[2], v.shape[2])
```

Comment:
I believe that doing `assert ... == ...` would work too.

Reply:
Oh, yeah, thanks for the reminder, @gabrieldemarmiesse